src/components/look/look-list/table/style-tag-list/index.tsx

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
+
+import { Button } from '@src/component/atoms';
+
+import { StyleTag } from '@src/types';
+import styleTagsData from '@src/data/style-tags';
+
+type StyleTagListProps = {
+  styleTags: StyleTag[];
+};
+
+const StyleTagList = ({ styleTags }: StyleTagListProps) => {
```

Comment:
```suggestion
function StyleTagList({ styleTags }: StyleTagListProps) {
```

컴포넌트는 function으로 선언합니다. React 컴포넌트 작성요령 노션 문서 확인해주세용

Reply:
686ca00